src/mock.rs

```diff
@@ -155,7 +153,10 @@ impl tellor::Config for Test {
 	type WeightToFee = ConstU128<10_000>;
 	type Xcm = TestSendXcm;
 	type XcmFeesAsset = XcmFeesAsset;
-	type XcmWeightToAsset = ConstU128<50_000>; // Moonbase Alpha: https://github.com/PureStake/moonbeam/blob/f19ba9de013a1c789425d3b71e8a92d54f2191af/runtime/moonbase/src/lib.rs#L135
+	type XcmWeightToAsset = ConstU128<50_000>;
+	#[cfg(feature = "runtime-benchmarks")]
+	type BenchmarkHelper = TestBenchmarkHelper;
+	type WeightInfo = (); // Moonbase Alpha: https://github.com/PureStake/moonbeam/blob/f19ba9de013a1c789425d3b71e8a92d54f2191af/runtime/moonbase/src/lib.rs#L135
```

Comment:
Comment on line 159 belongs to `XcmWeightToAsset` and should therefore follow that associated type definition.